### PR TITLE
QA Suggestions

### PR DESCRIPTION
**Pull request by jeff-boykin**
- [ ] Codecademy link is broken. Plus, where should it link to specifically?
- [ ] Should add links to things like DASH or downloads on the Prerequisites section

### CODE REVIEW

**Comment by melodyserra** [comment]
Can you please specify where the Codecademy link is broken for me?

File: curriculum/02-materials/readme.md [hunk 1]
```diff
@@ -20,17 +20,17 @@ creator:
 ### STUDENT PRE-WORK
 *Before this lesson, you should:*
 
-- Install Sublime Text 3 on your computer.
-- Have Google Chrome on your computer.
-- Complete DASH Project 1.
+- Install [Sublime Text 3](#) on your computer.
```
Comment (jeff-boykin) [inline curriculum/02-materials/readme.md]: - [ ] I'd add links for these here to stay consistent and make it easier for instructors.

File: curriculum/01-onboarding/03-roadmap.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ In order to make sure that students are best prepared for this workshop, we sugg
 In order to provide students with continued pathways for growth, we suggest using the following resources to create a custom learning plan, based on your students' goals and interests:
 
 * [DASH](dash.ga.co)
-* [Codecademy](codecademy.com)
+* [Codecademy](www.codecademy.com)
```
Comment (jeff-boykin) [inline curriculum/01-onboarding/03-roadmap.md]: - [ ] Codecademy has a ton of tutorials. Are there any in particular that they should be looking for? I'd add that as a note and/or link to those here.

File: curriculum/02-materials/readme.md [hunk 3]
```diff
@@ -55,56 +55,66 @@ creator:
...
+- Back End:  everything the user does not see, but is *affected by*. This involves the database (which is basically a virtual spreadsheet that stores data), the server (a machine/computer that hosts web files), and any application logic (such as the validation of their username and password).
 
-We'll mostly be focusing on JavaScript as a front end language in this lesson. So let's break down the THREE front end languages - HTML, CSS, and JavaScript.
+We'll mostly be focusing on JavaScript as a **front end language** in this lesson. So let's break down the THREE front end languages - HTML, CSS, and JavaScript.
 
 - HTML = noun (responsible for the 'things' on the page, e.g image vs paragraph)
 - CSS = adjective (responsible for styles like color or font family)
 - JavaScript = verb (responsible for behaviors, like drop-down menus)
 
+> Instructor Note: Some students may ask about Javascript as a back end language. Table these questions for now and return to them at the end of the session.
```
Comment (jeff-boykin) [inline curriculum/02-materials/readme.md]: - [ ] Instructor Notes help troubleshoot or problem solve common questions/issues by explaining the purpose or logic behind the discussion of a topic. I've added a preemptive one here but these should be included throughout.

File: curriculum/02-materials/readme.md [hunk 4]
```diff
@@ -55,56 +55,66 @@ creator:
...
+
 ### Activity
 
-Let's go to [lyft.com](https://www.lyft.com/). When you hover over the word “Explore” in the navigation menu, or notice the dynamically looping hero images, you are seeing JavaScript in action. You can also choose another website like [yelp.com](http://www.yelp.com/) or [airbnb.com](https://www.airbnb.com/). Answer the following question:
-- Which parts of the site are front end and which is back end?
+Let's go to [lyft.com](https://www.lyft.com/). When you hover over the word “Explore” in the navigation menu, or notice the dynamically looping hero images, you are seeing JavaScript in action. You can also choose another website like [yelp.com](http://www.yelp.com/) or [airbnb.com](https://www.airbnb.com/). 
+
+Looking at these sites, answer the following questions:
```
Comment (jeff-boykin) [inline curriculum/02-materials/readme.md]: - [ ] How should this activity be run? We could define this more clearly for instructors. E.g. Should they break into pairs or groups and compare their thoughts on the same site? Different sites?

File: curriculum/02-materials/readme.md [hunk 5]
```diff
@@ -116,38 +126,41 @@ So what exactly is JavaScript and what is it responsible for? To summarize, Java
...
+> Instructor Note: When students finish, have one or two pairs share out their understanding of what is going on and what the JavaScript is doing. Then ask another pair to share their solution to changing the slow button bulb to yellow, and fixing the go button.
 
 ## Demo: Pseudo Code (5 min)
 
-Before we start writing any Javascript, we'll learn a bit about how to plan out your code by writing pseudocode. What is pseudocode? Well, to put in simple terms, it's a way to 'plan out' your program in plain English before coding it, which will make the coding process infinitely more seamless! It will serve as a detailed yet readable description of what a computer program must do.
+Before we start writing any Javascript, we'll learn a bit about how to plan out your code by writing pseudocode. What is pseudocode? Well, to put in simple terms, it's a way to 'plan out' your program in plain English before coding it. This will make the coding process infinitely more seamless! Pseudocode serves as a detailed yet readable description of what a computer program must do.
```
Comment (jeff-boykin) [inline curriculum/02-materials/readme.md]: - [ ] Doesn't really matter if we spell this as one word or two, but should be consistent regardless.

File: curriculum/02-materials/readme.md [hunk 6]
```diff
@@ -263,15 +286,19 @@ typeof 3.45;
 typeof true;
 ```
 
-What data types did we store as the variable? That's right string and numbers. We'll learn about Booleans later in this class.
-
+<details>
+<summary>
+**Check:** What data types did we store as the variable? 
+</summary>
+That's right: string and numbers! We'll also learn about *Booleans* later in this class.
```
Comment (jeff-boykin) [inline curriculum/02-materials/readme.md]: - [ ] It's a best practice to highlight new terms the first time they show up, so as to better alert students and remind instructors to define them.

File: curriculum/02-materials/readme.md [hunk 7]
```diff
@@ -300,30 +327,47 @@ num -= 6
 => 2
 ```
 
-You can use ++ and -- to increment and decrement by 1, respectively. These can be used as prefix or postfix operators.
+You can use `++` and `--` to increment and decrement by 1, respectively. These can be used as *prefix* or *postfix* operators.
```
Comment (jeff-boykin) [inline curriculum/02-materials/readme.md]: - [ ] Are you sure they'll understand what these terms mean by context alone? Would be better to follow up with a knowledge check to define them (e.g. "What is a prefix operator? Can you create an example?" etc)

File: curriculum/02-materials/readme.md [hunk 8]
```diff
@@ -546,12 +602,14 @@ We will now transition in to the last part of the jQuery-code-along project. The
...
+
 So, who needs to know about this stuff? Everyone!  Even if you’re not the one doing the coding, being able to communicate effectively with developers and understand what’s going on behind the scenes of a product is a very important and highly valued skill in today’s job market.  
 
-A couple last things. You are going to get lots of errors…
+A couple of last things. You are going to get lots of errors. Don't get upset! This happens to everyone. Instead, try:
+
 - Google first, ask questions later (but don't be afraid to ask)
-- Stack Overflow is great for this stuff!
-- Get immersed!  Coding is hard if you don't get really involved.
+- [Stack Overflow](#) is great for this stuff!
```
Comment (jeff-boykin) [inline curriculum/02-materials/readme.md]: - [ ] Should link to an example here

**Comment by jeff-boykin** [comment]
@jeannyvaidya Completed my QA pass. See diff for edit suggestions and comments.

File: curriculum/01-onboarding/03-roadmap.md (hunk 2 above)
Comment (melodyserra) [inline curriculum/01-onboarding/03-roadmap.md]: This is just a recommendation on where to find additional resources - I think that which tutorials students decide on will depend, if they enjoyed the front end maybe they will try the html and css one, if they want to dig in to back end maybe they will try python or ruby. This is the reason for which I decided to keep it broad. Thoughts?

File: curriculum/02-materials/readme.md (hunk 8 above)
Comment (melodyserra) [inline curriculum/02-materials/readme.md]: Same comment here that I had about codecademy, what the students do with the resource really depends on their interests. Let me know if you disagree.

**Comment by melodyserra** [comment]
I just pushed some of the changes you recommended to master. Let me know if you still think more needs to be done. I don't really know which link is broken, let me know.

**Comment by jnappy** [comment]
Hey @melodyserra - resolved these merge conflicts and it looks great!  One thing to clear up: for the guided activities, are students loading the code in the browser after completing the challenge in sublime?  If so, it would be good to point that out in the directions and ask instructors to walk students through how to load a file in sublime and then open it in chrome for the first exercise.

**Comment by melodyserra** [comment]
@jnappy - Hi! Yes, students would be opening up code in browser. I will add that to the lesson. Which branch are you referring to? Master?

**Comment by jnappy** [comment]
Awesome, great - could you add to the `jeff-boykin-QA-1` branch?  Thanks!
